src/lists.py

```diff
@@ -8,22 +8,29 @@
 
 # Change x so that it is [1, 2, 3, 4]
 # YOUR CODE HERE 
+x.append(4)
 print(x)
 
 # Using y, change x so that it is [1, 2, 3, 4, 8, 9, 10]
 # YOUR CODE HERE 
+x.extend(y)
 print(x)
 
 # Change x so that it is [1, 2, 3, 4, 9, 10]
 # YOUR CODE HERE 
+del x[4]
```

Comment:
Interesting choice. You could also do `remove(8)` to remove the first occurrence of 8, or `pop(4)` to remove the fourth index.